src/components/AboutPage/AboutPage.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as styles from '../../styles/AboutPage.module.css';
+import { useNavigate } from 'react-router-dom';
+
+function AboutPage(props) {
+  const navigate = useNavigate();
+
+  const aboutHeading = <h1>About us</h1>;
+  const aboutTxt = 'We have implemented this app to give access to people from all across the globe to the current and also future weather forecast for cities in any country. With the help of few clicks you can find information about the temperature, humidity, pressure, visibility, wind, as well as other weather characteristics for your chosen city of interest.';
+
+  const navigateToHome = () => navigate('/');
+
+  return (
+    <>
```

Comment:
here is a good example where a div instead of a fragment would probably be a better fit since you're wrapping an entire page